src/include/platform/CHIPDeviceConfig.h

```diff
@@ -136,6 +136,16 @@
 #define CHIP_DEVICE_CONFIG_DEVICE_VENDOR_ID 9050
 #endif
 
+/**
+ * CHIP_DEVICE_CONFIG_USE_TEST_SERIAL_NUMBER
+ *
+ * Enables the use of a hard-coded default serial number if none
+ * is found in Chip NV storage.
+ */
+#ifndef CHIP_DEVICE_CONFIG_USE_TEST_SERIAL_NUMBER
```

Comment:
nit: naming of `...USE_TEST_...` makes it sound as a bool when this is actually a string. How about dropping `USE` in the name?

Reply:
Sure, it's actually a common directive used in other projects, i will rename them all